src/components/button/button.stories.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { text, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
-import { dlsThemeSelector, classicThemeSelector } from '../../../.storybook/theme-selectors';
+import { dlsThemeSelector } from '../../../.storybook/theme-selectors';
```

Comment:
do we need that `dlsThemeSelector`? Looks like it's an empty object.

Reply:
We do not. Resolved in a8a2ac277d632d9784b013f3cec0876fd99af73f